### ISSUE TITLE
Dependency seld/jsonlint is missing in current master branch (v1.8)

### ISSUE THREAD

**Issue by gplanchat** [akeneo/pim-community-dev#6687]
`seld/jsonlint` is missing in the current build, webpack build does not work.

This is the log of command `npm install && npm run webpack`
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/7.1.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'webpack' ]
2 info using npm@3.10.9
3 info using node@v7.1.0
4 verbose run-script [ 'prewebpack', 'webpack', 'postwebpack' ]
5 info lifecycle pim-community-standard@1.8.0~prewebpack: pim-community-standard@1.8.0
6 silly lifecycle pim-community-standard@1.8.0~prewebpack: no script for prewebpack, continuing
7 info lifecycle pim-community-standard@1.8.0~webpack: pim-community-standard@1.8.0
8 verbose lifecycle pim-community-standard@1.8.0~webpack: unsafe-perm in lifecycle true
9 verbose lifecycle pim-community-standard@1.8.0~webpack: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/gplanchat/Perso/projects/pim-community-standard/node_modules/.bin:/usr/local/Cellar/node/7.1.0/bin:/Users/gplanchat/.rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/local/bison/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/MacGPG2/bin:/Users/gplanchat/.composer/vendor/bin
10 verbose lifecycle pim-community-standard@1.8.0~webpack: CWD: /Users/gplanchat/Perso/projects/pim-community-standard
11 silly lifecycle pim-community-standard@1.8.0~webpack: Args: [ '-c',
11 silly lifecycle   'webpack --display-error-details --config vendor/akeneo/pim-community-dev/webpack.config.js --progress --display-modules' ]
12 silly lifecycle pim-community-standard@1.8.0~webpack: Returned: code: 2  signal: null
13 info lifecycle pim-community-standard@1.8.0~webpack: Failed to exec webpack script
14 verbose stack Error: pim-community-standard@1.8.0 webpack: `webpack --display-error-details --config vendor/akeneo/pim-community-dev/webpack.config.js --progress --display-modules`
14 verbose stack Exit status 2
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:885:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid pim-community-standard@1.8.0
16 verbose cwd /Users/gplanchat/Perso/projects/pim-community-standard
17 error Darwin 16.7.0
18 error argv "/usr/local/Cellar/node/7.1.0/bin/node" "/usr/local/bin/npm" "run" "webpack"
19 error node v7.1.0
20 error npm  v3.10.9
21 error code ELIFECYCLE
22 error pim-community-standard@1.8.0 webpack: `webpack --display-error-details --config vendor/akeneo/pim-community-dev/webpack.config.js --progress --display-modules`
22 error Exit status 2
23 error Failed at the pim-community-standard@1.8.0 webpack script 'webpack --display-error-details --config vendor/akeneo/pim-community-dev/webpack.config.js --progress --display-modules'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the pim-community-standard package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     webpack --display-error-details --config vendor/akeneo/pim-community-dev/webpack.config.js --progress --display-modules
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs pim-community-standard
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls pim-community-standard
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

I fixed this with `composer require seld/jsonlint`

**Comment by aRn0D** [comment]
Hi Grégory, 

Thank you for raising this issue. I am not a front developer but I send them your issue.

**Comment by gplanchat** [comment]
It seems to be solved, no more issue on ~1.8~ 2.0
